projects/igniteui-angular/src/lib/core/styles/typography/_base.scss

```diff
@@ -9,6 +9,7 @@
 /// @param {String} $font-family [null] - The font size of the type style.
```

Comment:
not related to this PR but this should be `The font family of the type style` IMO